spanner/jdbc/src/main/java/com/example/spanner/jdbc/AutocommitUpdateDataExample.java

```diff
@@ -45,8 +45,8 @@ static void update(String projectId, String instanceId, String databaseId) throw
       // The following statement will automatically be committed by the connection.
       int updateCount =
           statement.executeUpdate(
-              "INSERT INTO Singers (SingerId, FirstName, LastName)\n"
-                  + "VALUES (9999, 'Lloyd', 'Pineda')");
+              "INSERT INTO Singers (SingerId, FirstName, LastName, Revenues)\n"
```

Comment:
Why do you add the \n ?

Reply:
(It's not specifically added in this change, so I assume that it is more of a general question for all these samples)

No really specific reason, other than that it is quite customary to have the `VALUES (...)` clause on a new line. See for example the examples here: https://cloud.google.com/spanner/docs/dml-syntax#insert_examples